lms/djangoapps/teams/templates/teams/teams.html

```diff
@@ -42,6 +42,7 @@
         teamsDetailUrl: '${ teams_detail_url }',
         teamMembershipsUrl: '${ team_memberships_url }',
         teamMembershipDetailUrl: '${ team_membership_detail_url }',
+        leaveTeamUrl: '${leave_team_url}',
```

Comment:
Nit: all the other usages here have spaces around the brackets, so you should probably stay consistent (I personally prefer not to include the whitespace)